object-manipulation-2/src/defaults.js

```diff
@@ -0,0 +1,11 @@
+/* exported defaults */
+function defaults(target, source) {
+  for (var property in source) {
+    for (var property2 in target) {
+      if (property !== property2) {
+        Object.assign(source, target);
+      }
+    }
+  }
```

Comment:
This can be stripped down to only using one loop. If `target` at `property` is `undefined`, then assign `source` at `property` to `target` at property.

Reply:
Thanks for the suggestion here! Your way is obviously so much simpler. It took me forever to figure out how to use Object.assign to copy all the right properties, but not overwrite existing properties. Def makes sense that I shouldn't have even been using it...